source/user-manual/agents/remote-upgrading/create-custom-wpk/generate-wpk-package-manually.rst

```diff
@@ -14,122 +14,122 @@ A WPK package must contain an installation program in binary form or a script in
  * Write a file called upgrade_result containing a status number (0 means OK) before exiting.
 
 Requirements
-^^^^^^^^^^^^^
+^^^^^^^^^^^^
 
  * Python 2.7 or 3.5+
  * The Python ``cryptography`` package. This may be obtained using the following command:
 
-    .. code-block:: console
+.. code-block:: console
 
-            pip install cryptography
+ pip install cryptography
 
 Linux WPK
-^^^^^^^^^^^
+^^^^^^^^^
 
 Install development tools and compilers. In Linux this can easily be done using your distribution's package manager:
 
  a) For RPM-based distributions:
 
-    .. code-block:: console
+.. code-block:: console
 
-            # yum install make gcc policycoreutils-python automake autoconf libtool unzip
+ # yum install make gcc policycoreutils-python automake autoconf libtool unzip
 
  b) For Debian-based distributions:
```

Comment:
![image](https://user-images.githubusercontent.com/3830293/67090838-6b18cd80-f1ab-11e9-8aff-3458a38212f2.png)
An example of wrong indentation